tests/flag_util_test.py

```diff
@@ -132,5 +133,18 @@ def testReadAndWrite(self):
     self.assertFlagState(flag_values_copy, 2, True)
 
 
+class TestUnitsParser(unittest.TestCase):
+  def setUp(self):
+    self.up = flag_util.UnitsParser()
+
+  def testParser(self):
+    self.assertEqual(self.up.Parse('10KiBo'),
```

Comment:
In Pint 0.6 (the current release), bytes are abbreviated `Bo`. In Pint 0.7 (the development branch), bytes are abbreviated `B`. I've been trying to figure out when there will be an official 0.7 release so we don't have to worry about this. In the meantime, I will introduce my own custom bytes unit with abbreviation `B`, ignore the built-in one, and standardize on `B`. That matches the rest of the world.